src/bakery_manager_mobile/lib/pages/edit_task_page.dart

```diff
@@ -24,39 +26,35 @@ class _EditTaskPageState extends State<EditTaskPage> {
 
   Task? task; // Variable to store the Task object
   Recipe? selectedRecipe;
+  Account? selectedAccount; // Variable to store selected Account
 
   // Recipe list for dropdown
   List<Recipe> _allRecipes = [];
-  List<Recipe> _filteredRecipes = [];
+
+  // Account list for dropdown
+  List<Account> _allAccounts = [];
+
+  @override
+  void didChangeDependencies() {
+    super.didChangeDependencies();
+    // Retrieve the Task object passed from the previous screen
+    task = ModalRoute.of(context)!.settings.arguments as Task;
+
+    // Prepopulate form fields with task data
+    amountToBakeController.text = task!.amountToBake.toString();
+    dueDateController.text = DateFormat('yyyy-MM-dd').format(task!.dueDate);
```

Comment:
the UI isn't allowing me to change the date or time???

Reply:
Fixed I am slowly loosing my ability to think